src/ServiceMonitor/resource.h

```diff
@@ -0,0 +1,19 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
+// Used by ServiceMonitor.rc
+//
+#define SM_MAJORNUMBER                  2
```

Comment:
These version settings are already defined in version.h. We should be using the values from version.h so this file may not be needed.